app/templates/teachers/edit-student-modal.jade

```diff
@@ -7,28 +7,36 @@ block modal-header-content
 
 block modal-body-content
 
-  - var status = view.user.prepaidStatus()
+  - var status = view.user.prepaidNumericalCourses()
   - var prepaidType = view.user.prepaidType('includeCourseIDs')
 
   .text-center
     .license-details
       strong(data-i18n="teacher.license_status")
       span.spr :
-      if status == "not-enrolled"
+      if status == 0
```

Comment:
Similar comment about `status` name, `status` of `0` is confusing. Also, `===` for comparing with `0`.